cmd/minishift/cmd/logs.go

```diff
@@ -48,6 +49,7 @@ var logsCmd = &cobra.Command{
 }
 
 func init() {
-	logsCmd.Flags().BoolVarP(&follow, "follow", "f", false, "Continously print the logs entries.")
+	logsCmd.Flags().BoolVarP(&follow, "follow", "f", false, "Continuously print the logs entries.")
+	logsCmd.Flags().Int64VarP(&tail, "tail", "t", -1, "Number of lines to show from the end of the logs (default -1 indicates show all lines)")
```

Comment:
@budhram Now if you compile and get the help section you get below which doesn't look good just remove ` (default -1 indicates show all lines)` part since you are using the `Int64VarP` this will automatic append default value for you.

```
$ ./minishift logs -h
Gets the logs of the running OpenShift cluster. The logs do not contain information about your application code.

Usage:
  minishift logs [flags]

Flags:
  -f, --follow     Continuously print the logs entries.
  -h, --help       help for logs
  -t, --tail int   Number of lines to show from the end of the logs (default -1 indicates show all lines) (default -1)
```

Reply:
@praveenkumar Latest changes doesn't say this. You seems to have running one commit older.
See https://github.com/minishift/minishift/pull/2830#issuecomment-425019094

Reply:
My bad, I didn't push the changes :)